File: 2525. Categorize Box According to Criteria/solution.py
class Solution(object):
  def categorizeBox(self, length, width, height, mass):
    """
    :type length: int
    :type width: int
    :type height: int
    :type mass: int
    :rtype: str
    """
    bulky = (length >= 10000) or (width >= 10000) or (height >= 10000) or (mass >= 10000) or ((length * width * height) >= 1000000000)
    heavy = mass >= 100
    
    if bulky and heavy:
      return "Both"
    elif not bulky and not heavy:
      return "Neither"
    elif bulky and not heavy:
      return "Bulky"
    elif not bulky and heavy:
      return "Heavy"
    
    return None

File: 2525. Categorize Box According to Criteria/solution.cpp
class Solution {
public:
  string categorizeBox(int length, int width, int height, int mass) {
    bool bulky = (length >= 10000) || (width >= 10000) || (height >= 10000) || (mass >= 10000) || (((long long) length * (long long) width * (long long) height) >= 1000000000);
    bool heavy = mass >= 100;
    
    if (bulky && heavy){
      return "Both";
    }
    else if (!bulky && !heavy){
      return "Neither";
    }
    else if (bulky && !heavy){
      return "Bulky";
    }
    else if (!bulky && heavy){
      return "Heavy";
    }
    
    return NULL;
  }
};